rllib/policy/policy.py

```diff
@@ -740,8 +740,44 @@ def get_state(self) -> PolicyState:
             # The current global timestep.
             "global_timestep": self.global_timestep,
         }
+        if self.config.get("enable_connectors", False):
```

Comment:
Can you add a one-line comment here? that we are adding the connector state?

Reply:
sure